webpack.config.js

```diff
@@ -164,25 +165,18 @@ module.exports = (env = {}) => {
         },
         {
           test: /\.(woff|ttf|eot|ico)/,
-          use: [
-            {
-              loader: 'file-loader',
-              options: {
-                name: '[name].[hash].[ext]',
-              },
-            },
-          ],
+          type: 'asset/resource',
+          generator: {
+            filename: '[name].[contenthash][ext]',
+          },
         },
-
         {
           test: /\.(svg|png|jpe?g|gif)/,
+          type: 'asset/resource',
+          generator: {
+            filename: '[name].[contenthash][ext]',
```

Comment:
I assume you no longer need to add  the dot between the contenthash and ext

Reply:
Correct, it's not needed anymore

https://webpack.js.org/guides/asset-modules/#custom-output-filename